CorsixTH/Lua/map.lua

```diff
@@ -841,4 +841,9 @@ function Map:afterLoad(old, new)
       MaxSalary = 2000,
     }
   end
+  if old < 187 then
+    self.level_config.gbv.Tired = 600
+    self.level_config.gbv.VeryTired = 700
+    self.level_config.gbv.CrackUpTired = 800
```

Comment:
Original game already has values, probably shouldn't overwrite them.
```suggestion
    self.level_config.gbv.Tired = self.level_config.gbv.Tired or 600
    self.level_config.gbv.VeryTired = self.level_config.gbv.VeryTired or 700
    self.level_config.gbv.CrackUpTired = self.level_config.gbv.CrackUpTired or 800
```